packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -232,6 +234,11 @@ class Overlay extends StatefulWidget {
   /// To remove an entry from an [Overlay], use [OverlayEntry.remove].
   final List<OverlayEntry> initialEntries;
 
+  /// {@macro flutter.widgets.Clip}
+  ///
+  /// Defaults to [Clip.none], and must not be null.
```

Comment:
remember to change documentation too when the default is changed to `Clip.hardEdge`.